CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ## Unreleased - 2019-??-??
 
+* Replace Collection addAll by constructor
```

Comment:
I'm not sure that this really needs to be in the changelog since it's invisible from the outside.

Reply:
I just followed the template. I am going to remove this changelog.